tools/rc/ResourceCompiler.cpp

```diff
@@ -795,6 +796,11 @@ checkSanity(option::Parser& parse, option::Option* options)
 int
 main(int argc, char** argv)
 {
+    // Deterministic build things to set
```

Comment:
I would prefer a comment that explains why the local is set in such a way for deterministic builds. This is not a helpful comment.